lighthouse-core/runner.js

```diff
@@ -130,6 +130,9 @@ class Runner {
         }
 
         return {
+          meta: {
+            lighthouseVersion: require('../package').version
```

Comment:
Not a huge deal, but I'd personally prefer this as a top level property. Seems as important as the URLs, for instance